WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormTableViewDataSource.swift

```diff
@@ -277,13 +279,31 @@ private extension ProductFormTableViewDataSource {
         cell.hideSeparator()
     }
 
+    func configurePromoteWithBlaze(cell: UITableViewCell) {
+        guard let cell = cell as? LeftImageTableViewCell else {
+            fatalError("Unexpected table view cell for the promote with Blaze cell")
+        }
+        let title = NSLocalizedString(
+            "productFormTableViewDataSource.promoteWithBlazeButton",
+            value: "Promote with Blaze",
+            comment: "Label for button on product form to open Blaze flow"
+        )
+
+        cell.configure(
+            image: .blaze.resize(to: Constants.blazeButtonIconSize)!,
```

Comment:
❓ Is it acceptable here to use `!` ? 

This assumes two things:
- that .blaze should exist,
- that the resizing works

Sometimes we do this in Android but would love to learn what's usually done in iOS.

Reply:
We try to avoid forced unwrapping where we can. How about accepting the image as optional in the `configure` method? Since the image view doesn't require the image, that would be best.

Reply:
Awesome, makes a lot of sense. Handled in https://github.com/woocommerce/woocommerce-ios/pull/10970/commits/97d4cb4f68dd2ee492dad411659558fa569a88fd